.circleci/config.yml

```diff
@@ -0,0 +1,127 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/circleci-champaign-ak-processor
+  docker:
+    - image: soutech/ruby2.4.1_node_awscli:0.2
+
+jobs:
+  install_dependencies:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          name: Restoring bundler cache
+          key: gem-cache-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Install gems
+          command: bundle install --jobs 4 --deployment
```

Comment:
`--deployment` just errors if there's an incompatibility between Gemfile and Gemfile.lock and defaults the installation to `vendor/bundle`